x-pack/platform/plugins/shared/onechat/public/application/routes.tsx

```diff
@@ -29,6 +31,17 @@ export const OnechatRoutes: React.FC<{}> = () => {
           <OnechatToolsPage />
         </Route>
       )}
+
+      {isAgentPageEnabled && (
```

Comment:
Since https://github.com/elastic/kibana/pull/226535, no longer need specific per-page feature flags - we can remove the `isAgentPageEnabled` logic